### DISCUSSION TITLE
ServiceMesh/The-future-of-ServiceMesh

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#221] [General]
# ServiceMesh/The-future-of-ServiceMesh

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/ServiceMesh/The-future-of-ServiceMesh.html

**Comment by EdAndrew** [comment]
如此之复杂的设计，真的有实用价值吗

**Reply by isno** [reply]
有用，ebpf 的服务网格是一些技术服务公司（Isovalent、Datadog）的拳头产品。它们在性能、安全上还是有不少优势的，海外的一些零售、金融巨头用的比较多。

**Reply by echo1937** [reply]
k8s本身已经很复杂了，叠加服务网格更复杂了。
